src/syft/lib/sympc/__init__.py

```diff
@@ -53,7 +56,13 @@ def create_ast(client: TypeAny = None) -> Globals:
 
     methods: TypeList[TypeTuple[str, str]] = [
         ("sympc.store.CryptoStore.get_primitives_from_store", "syft.lib.python.List"),
+        (
+            "sympc.store.CryptoStore.populate_store",
```

Comment:
Q: Could you remove this? It was already added by @arturomf94 (it is at line 73) :D

Reply:
OK!